services/demo/src/view/forms/ComplexForm.tsx

```diff
@@ -13,17 +13,23 @@ import {
 } from '@react-declarative-form/material-ui';
 import { Grid, FormControlLabel, Switch } from '@material-ui/core';
 
-const lessThanCurrencyField = (targetFieldName: keyof ComplexFormFields) => (
+const lessThanCurrencyField = (
+    targetFieldName: keyof ComplexFormFields,
+    value: string,
```

Comment:
Why do we need to pass in the value here?

Reply:
the values here are the NestedFrom values, not the parent From values, so it could not locate the values[targetFieldName]